README.rst

```diff
@@ -104,36 +104,30 @@ If you're using `environments <http://conda.pydata.org/docs/intro.html>`_
 managed through ``conda`` (recommended), this will
 get you started: ::
 
-    conda create --name=cloudside python=3.6 notebook pytest pandas matplotlib requests coverage
-
-Followed by: ::
-
-    conda activate cloudside
-    conda install metar --channel=conda-forge
+    conda create --name=cloudside python=3.8 notebook pytest pandas matplotlib requests coverage
```

Comment:
since we are testing against pip install metar and there are no fancy dependency binaries, why would we advise users to install via conda and/or conda-forge?

Reply:
it seems that where projects _can_ be pure pip, they should be and the tests prove that users don't need conda-forge to make this work.